codes/quantum/qubits/bacon_shor.yml

```diff
@@ -8,8 +8,17 @@ physical: qubits
 logical: qubits
 
 name: 'Bacon-Shor code'
-#introduced: '\cite{arxiv:quant-ph/0501184}'
+introduced: '\cite{doi:10.1103/PhysRevA.73.012340}'
 
-description: 'Stub.'
+description: 'A subsytem code defined on a\(2D\) lattice with the parameters \([n^2,1,n]\) and on a \(3D\) lattice with the parameters \([n^3,1,n]\) motivated by Hamiltonians defined on square and cubic lattices with anisotropic spin-spin interactions. The shortest [9,1,3] Bacon-Shor code has 6 gauge operators, symmetric in both X and Z, as compared to the 8 stabilizers of Shor''s \([9,1,3]\) code.'
```

Comment:
I'd like to define these so that they include code_id:quantum_parity, which means that this entry needs to include the asymmetric 2D bacon-shor codes [[m_1 m_2, 1, \min(m_1, m_2)]]. Ted Yoder's thesis describes these pretty well: https://dspace.mit.edu/handle/1721.1/115680. I suppose you could generalize the 3D case to asymmetric blocks as well, but this is not necessary.